core/src/test/scala/org/apache/spark/SparkThrowableSuite.scala

```diff
@@ -36,6 +38,15 @@ import org.apache.spark.util.Utils
  */
 class SparkThrowableSuite extends SparkFunSuite {
 
+  /* Used to regenerate the error class file. Run:
+   {{{
+      SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "core/testOnly *SparkThrowableSuite"
```

Comment:
```suggestion
      SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "core/testOnly *SparkThrowableSuite -- -t 'Error classes are correctly formatted'"
```

Reply:
Updated - I couldn't get the single quotes to work, but was able to escape with backslashes.